config/webpack.server.js

```diff
@@ -48,7 +48,7 @@ module.exports = (env, argv) => {
       new FilterWarningsPlugin({
         // suppress conflicting order warnings from mini-css-extract-plugin.
         // see https://github.com/webpack-contrib/mini-css-extract-plugin/issues/250
-        exclude: /Conflicting order between:/,
+        exclude: /Conflicting order. Following module has been added/,
```

Comment:
:see_no_evil: If we cannot see the warning, is there even a problem? :thinking:

Reply:
The warning doesn't really matter that much to as (we could fix it but eh :shrug:). And it's super annoying with all the spam every time you compile the app.

Reply:
Yeah. Even if we were to fix this, it would be better to do it in a separate PR.